packages/store-api/package.json

```diff
@@ -14,19 +14,24 @@
   },
   "scripts": {
     "develop": "tsdx watch",
-    "build": "tsdx build",
+    "build": "graphql-codegen --config codegen.yml && tsdx build",
     "test": "tsdx test",
-    "lint": "tsdx lint"
+    "lint": "tsdx lint",
+    "generate": "graphql-codegen --config codegen.yml"
```

Comment:
How about adding a `clean` script here too?

Reply:
for cleaning the `dist` folder you mean?

Reply:
I was actually thinking about the generated types... but yeah! Hahahah the dist folder makes more sense

Reply:
Oh, every time we generate the typings it actually clean the types, so it's kinda already built into the "generate" command